daycare/test_daycare.py

```diff
@@ -14,6 +15,12 @@
 CITY = "Tel-Aviv"
 ADDRESS = "The best street 5"
 
+IMAGE_NAME1 = "Daycare yard"
+IMAGE1 = "../../static/images/doggiedaycare.jpeg"
+
+IMAGE_NAME2 = "Daycare patio"
+IMAGE2 = "../../static/images/dog-daycare-9dc7cf7477f748f7ac99cdee3564041b.jpg"
+
```

Comment:
I think the image files name should indicate that they are for testing and maybe even under test folder